editor/find_in_files.cpp

```diff
@@ -73,10 +68,10 @@ static bool find_next(const String &line, String pattern, int from, bool match_c
 		out_end = end;
 
 		if (whole_words) {
-			if (begin > 0 && is_text_char(line[begin - 1])) {
+			if (begin > 0 && (is_ascii_identifier_char(line[begin - 1]))) {
 				continue;
 			}
-			if (end < line.size() && is_text_char(line[end])) {
+			if (end < line.size() && (is_ascii_identifier_char(line[end]))) {
```

Comment:
Strictly speaking the parentheses are redundant here (but that's not a big deal :)).

Reply:
Yeah don't bother, it's fine as is :)
There's other places where it was already using this style.